physionet-django/console/templates/console/event_management.html

```diff
@@ -46,11 +46,51 @@ <h1>{{ event.title }}</h1>
             <div class="col-md-11">
               <button class="btn btn-sm btn-primary"
                       data-toggle="modal"
-                      data-target="#view-participants">View participants</button>
+                      data-target="#participants">View</button>
             </div>
           </div>
         </div>
       </div>
+      <div class="row mb-1">
+        <div class="col-md-3">Pending Applications:</div>
+        <div class="col-md-9">
+          <div class="row mb-1">
+            <div class="col-md-1">{{ event.participants.count }}</div>
```

Comment:
`even.participants.count` could be memoized using the `with` templatetag.

Reply:
I've updated the view to include the count alongwith the actual applications as suggested in another review snippet.